### ISSUE TITLE
accessibilityActions is causing problems with accessible props (Android - accessibility)

### ISSUE THREAD

**Issue by felire** [facebook/react-native#29715]
## Description
Hi! This is an easy problem to describe. 

We have this piece of code:
```
<View
      style={style.rangeContainer}
      {...accessibilityProps}
      // @ts-ignore
      onAccessibilityAction={onRangeAccessibilityAction}
    >
      <Text style={style.rangeTitle}>{label}</Text>
      <Text style={style.selectedValue}>{value}</Text>
    </View>
```

Where the `accessibilityProps` are these:

```
accessibilityActions: [
        {
          name: AccessibleSliderActions.decrement,
        },
        {
          name: AccessibleSliderActions.increment,
        },
      ],
      accessibilityHint: sliderHint(label, step),
      accessibilityLabel: sliderLabel(label, sliderValue, maximum),
      accessibilityRole: 'adjustable',
      accessible: true,
      accessibleValue: { max: maximum, min: minimum, now: sliderValue },
```

Me and me team found that `accessible` and `accessibilityActions` can't coexist. I mean, if I want to use this like it's described above. When I click on the first Text or on the second the screen reader will read that text instead of selecting the entire View, which is wrong because the accessible prop is set to **true**

## React Native version:
We are using `0.61.5`

## Steps To Reproduce
Those are explained above

## Expected Results
The expected result is that both props can coexist and everything works as it should. Those Text shouldn't be accessibles based on Docs.

**Comment by safaiyeh** [comment]
Hi @felire thanks for the issue. Sorry you're having trouble, could you verify this occurs on the latest version?

**Comment by felire** [comment]
@safaiyeh The bug is still happening on the lats version of RN. 

Tested on `0.63.2`

**Comment by vitorandrietta** [comment]
hi, this is also happening with in version `0.63.2`
after passing `accessibilityActions` prop to the view I become able to select its children elements, even after setting 
`accessible={true}` in the parent view

**Comment by amarlette** [comment]
This issue should be resolved by #30851 once that issue is closed as part of the Improved React Native Accessibility Project

**Comment by fabriziobertoglio1987** [comment]
a solution, as done with [Text onPress and onLongPress](https://github.com/facebook/react-native/pull/34284/files), is setting focusable={true} when prop `accessibilityAction != null`
